internal/bminventory/inventory_test.go

```diff
@@ -889,18 +889,23 @@ var _ = Describe("IgnitionParameters", func() {
 	})
 })
 
-func createCluster(db *gorm.DB, status string) *common.Cluster {
+func createClusterWithAvailability(db *gorm.DB, status string, highAvailabilityMode string) *common.Cluster {
 	clusterID := strfmt.UUID(uuid.New().String())
 	c := &common.Cluster{
 		Cluster: models.Cluster{
-			ID:     &clusterID,
-			Status: swag.String(status),
+			ID:                   &clusterID,
+			Status:               swag.String(status),
+			HighAvailabilityMode: &highAvailabilityMode,
 		},
 	}
 	Expect(db.Create(c).Error).ToNot(HaveOccurred())
 	return c
 }
 
+func createCluster(db *gorm.DB, status string) *common.Cluster {
```

Comment:
not sure i understand why it is needed?

Reply:
Most tests don't care about availability mode, this is the function they call. It was there before, the diff makes it look like I added it.